corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -3,7 +3,8 @@
 {% load i18n %}
 
 {% block js-inline %}{{ block.super }}
-{% include 'cloudcare/includes/touchforms-inline.html' %}
+    <script type="text/javascript" src="{% static 'moment/moment.js' %}"></script>
```

Comment:
a bit hacky that this is in `js-inline`. also i think it might be better to move this to touchforms-inline so other places that use it will get this import